recruiter-front-end/src/views/AddCandidatePage/AddCandidatePage.js

```diff
@@ -0,0 +1,190 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react';
+import { Grid, Divider, Form, Dropdown, Input } from 'semantic-ui-react';
+
+const flexContainer = {
+  display: 'flex',
+  flexDirection: 'column',
+};
+
+const center = {
+  textAlign: 'center',
+};
+
+const DegreeSelection = [
```

Comment:
Can you abstract away the DegreeSelection, MajorSelection, and ExperienceSelection into components and import them? Just to clear up the code